packages/core/src/UserState.ts

```diff
@@ -130,17 +186,17 @@ export default class UserState {
     }
 
     /**
-     * Query the latest user state transition epoch. If user hasn't performed user state transition,
-     * the function will return the epoch which user has signed up in Unirep contract.
+     * Query the current database for a user's signup event or latest user state transition [nullifier](https://developer.unirep.io/docs/protocol/nullifiers).
+     * @param attesterId The attester to be queried. Default: `this.attesterId`
      * @returns The latest epoch where user performs user state transition.
```

Comment:
'The latest epoch where _a_ user ~performs~ _performed a_ user state transition.'